test/util/Aliases.ts

```diff
@@ -89,11 +89,51 @@ export function double(value: string): string {
  * '2001-10-26T21:32:52' => "2001-10-26T21:32:52"^^xsd:dateTime
  * @param value string (representing a date)
  */
-export function date(value: string): string {
+export function dateTimeNotation(value: string): string {
   return compactTermString(value, 'xsd:dateTime');
 }
 
-function compactTermString(value: string, dataType: string): string {
+/**
+ * ''02:12:59'' => "'02:12:59'"^^xsd:time
+ * @param value string (representing a date)
+ */
+export function timeNotation(value: string): string {
```

Comment:
The longer function names seem too vague and ambiguous for their length. It's also not consistent with the other short ones like int and decimal. Is there a reason these ones have 'Notation' added?
How about these: `xsd_time(val)`, `xsdTime(val)`,  `typedTime(val)`, `timeTyped(val)`, `addTimeType(val)` or `withTimeType(val)`?

Reply:
I like the timeTyped option. :D